rocky/rocky/paginator.py

```diff
@@ -0,0 +1,38 @@
+from django.core.paginator import EmptyPage, Page, PageNotAnInteger, Paginator
+from django.utils.translation import gettext_lazy as _
+
+
+class RockyPaginator(Paginator):
+    def __init__(
+        self,
+        object_list,
+        per_page: int | str,
+        orphans: int = 0,
+        allow_empty_first_page: bool = True,
+    ) -> None:
+        super().__init__(object_list, per_page, orphans, allow_empty_first_page)
+
+        if orphans != 0:
+            raise ValueError("Setting orphans is not supported")
```

Comment:
Using `*args` and `**kwargs` would make this forward compatible with any further changes in Django such as the error_messages argument that is added in Django 5.0.

```suggestion
    def __init__(
        self,
        *args,
        **kwargs,        
    ) -> None:
        super().__init__(*args, **kwargs)

        if self.orphans != 0:
            raise ValueError("Setting orphans is not supported")
```